From: unknown author <unknown.author@example.com>
Date: Mon, 12 Sep 2022 08:39:41 +0200
Subject: [PATCH] auth/credentials/wscript_build: to remove dependency cmocka,
 remove tests

---
 auth/credentials/wscript_build | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/auth/credentials/wscript_build b/auth/credentials/wscript_build
index 7568554df4d6..401b6aaac594 100644
--- a/auth/credentials/wscript_build
+++ b/auth/credentials/wscript_build
@@ -35,9 +35,3 @@ bld.SAMBA_PYTHON('pycredentials',
     public_deps='samba-credentials %s %s CREDENTIALS_CMDLINE CREDENTIALS_KRB5 CREDENTIALS_SECRETS' % (pytalloc_util, pyparam_util),
     realname='samba/credentials.so'
 )
-
-bld.SAMBA_BINARY('test_creds',
-                 source='tests/test_creds.c',
-                 deps='cmocka samba-credentials',
-                 local_include=False,
-                 for_selftest=True)
