From: unknown author <unknown.author@example.com>
Date: Mon, 12 Sep 2022 08:39:41 +0200
Subject: [PATCH] lib/compression/wscript_build: to remove dependency cmocka,
 remove tests

---
 lib/compression/wscript_build | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/lib/compression/wscript_build b/lib/compression/wscript_build
index 61fe4a9808e6..aadb659530be 100644
--- a/lib/compression/wscript_build
+++ b/lib/compression/wscript_build
@@ -5,20 +5,6 @@ bld.SAMBA_SUBSYSTEM('LZXPRESS',
                     source='lzxpress.c lzxpress_huffman.c'
                     )
 
-bld.SAMBA_BINARY('test_lzx_huffman',
-                 source='tests/test_lzx_huffman.c',
-                 deps=('cmocka replace LZXPRESS'
-                       ' samba-util'),
-                 local_include=False,
-                 for_selftest=True)
-
-bld.SAMBA_BINARY('test_lzxpress_plain',
-                 source='tests/test_lzxpress_plain.c',
-                 deps=('cmocka replace LZXPRESS'
-                       ' samba-util'),
-                 local_include=False,
-                 for_selftest=True)
-
 bld.SAMBA_PYTHON('pycompression',
                  'pycompression.c',
                  deps='LZXPRESS',
