From: unknown author <unknown.author@example.com>
Date: Wed, 15 Nov 2023 13:48:46 +0100
Subject: [PATCH] lib/tevent/wscript: to remove dependency cmocka, remove tests

---
 lib/tevent/wscript | 44 --------------------------------------------
 1 file changed, 44 deletions(-)

diff --git a/lib/tevent/wscript b/lib/tevent/wscript
index a0c3876304a3..ceb4681685ad 100644
--- a/lib/tevent/wscript
+++ b/lib/tevent/wscript
@@ -19,7 +19,6 @@ from waflib import Options, Logs, Context, Errors
 
 samba_dist.DIST_DIRS('''lib/tevent:. lib/replace:lib/replace
                         lib/talloc:lib/talloc buildtools:buildtools
-                        third_party/cmocka:third_party/cmocka
                         third_party/waf:third_party/waf''')
 
 def options(opt):
@@ -33,14 +32,6 @@ def configure(conf):
     conf.RECURSE('lib/replace')
     conf.RECURSE('lib/talloc')
 
-    if conf.CHECK_FOR_THIRD_PARTY():
-        conf.RECURSE('third_party/cmocka')
-    else:
-        if not conf.CHECK_CMOCKA():
-            raise Errors.WafError('cmocka development package have not been found.\nIf third_party is installed, check that it is in the proper place.')
-        else:
-            conf.define('USING_SYSTEM_CMOCKA', 1)
-
     conf.env.standalone_tevent = conf.IN_LAUNCH_DIR()
 
     if not conf.env.standalone_tevent:
@@ -82,9 +73,6 @@ def build(bld):
     bld.RECURSE('lib/replace')
     bld.RECURSE('lib/talloc')
 
-    if bld.CHECK_FOR_THIRD_PARTY():
-        bld.RECURSE('third_party/cmocka')
-
     SRC = '''tevent.c tevent_debug.c tevent_fd.c tevent_immediate.c
              tevent_queue.c tevent_req.c tevent_wrapper.c
              tevent_poll.c tevent_threads.c
@@ -124,7 +112,6 @@ def build(bld):
                          realname='_tevent.so',
                          cflags='-DPACKAGE_VERSION=\"%s\"' % VERSION)
 
-
         bld.INSTALL_WILDCARD('${PYTHONARCHDIR}', 'tevent.py', flat=False)
 
         # install out various python scripts for use by make test
@@ -132,37 +119,6 @@ def build(bld):
                          pattern='tevent.py',
                          installdir='python')
 
-    bld.SAMBA_BINARY('test_tevent_tag',
-                     source='tests/test_tevent_tag.c',
-                     deps='cmocka tevent',
-                     install=False)
-
-    bld.SAMBA_BINARY('test_tevent_trace',
-                     source='tests/test_tevent_trace.c',
-                     deps='cmocka tevent',
-                     install=False)
-
-def test(ctx):
-    '''test tevent'''
-    print("The tevent testsuite is part of smbtorture in samba4")
-
-    samba_utils.ADD_LD_LIBRARY_PATH('bin/shared')
-    samba_utils.ADD_LD_LIBRARY_PATH('bin/shared/private')
-
-    pyret = samba_utils.RUN_PYTHON_TESTS(['bindings.py'])
-
-    unit_test_ret = 0
-    unit_tests = [
-        'test_tevent_tag',
-        'test_tevent_trace',
-    ]
-
-    for unit_test in unit_tests:
-        unit_test_cmd = os.path.join(Context.g_module.out, unit_test)
-        unit_test_ret = unit_test_ret or samba_utils.RUN_COMMAND(unit_test_cmd)
-
-    sys.exit(pyret or unit_test_ret)
-
 def dist():
     '''makes a tarball for distribution'''
     samba_dist.dist()
